components/crud-web-apps/tensorboards/Dockerfile

```
@@ -7,7 +7,7 @@ COPY ./common/backend/ .
 RUN python3 setup.py bdist_wheel
 
 # --- Build the frontend kubeflow library ---
-FROM node:16.20.2-bullseye as frontend-kubeflow-lib
+FROM node:24-bookworm as frontend-kubeflow-lib
```

for more _deterministic builds_ - I think we should fix this new version to a major/minor/patch version just like it was originally.

```suggestion
FROM node:24.11.1-bookworm as frontend-kubeflow-lib
```